### PR TITLE
chore: add coverage codecov

### PR DESCRIPTION
**Pull request by thedaviddias**
***Short description of what this resolves:***

Add Codecov support

***Proposed changes:***

Add codecov to Github actions

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,3 +41,14 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Run coverage
+        if: success() && matrix.os == 'ubuntu-latest' && matrix.node == '12.x'
```
Comment (Shinigami92) [inline .github/workflows/ci.yml:46]: I don't know if `success()` is correct here
What if some tests are failing? Is the report not running?
Is that intended?

Comment (thedaviddias) [reply .github/workflows/ci.yml]: Removed success

File: package.json [hunk 2]
```diff
@@ -108,5 +109,21 @@
...
+  ],
+  "nyc": {
+    "all": true,
+    "include": [
+      "dist/htmlhint.js"
+    ],
+    "exclude": [
+      "**/*.spec.js",
+      "src/**/*.js",
+      "bin/**/*.js"
```
Comment (Shinigami92) [inline package.json:121]: `bin/**/*.js` will not exist in `beta` anymore
There is only `bin/htmlhint` (without any extension)
And this points to `dist/cli/htmlhint.js`

Should we move the target to `beta`?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -46,3 +47,14 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Run coverage
```
Comment (Shinigami92) [inline .github/workflows/ci.yml:51]: I'm not quite sure if this part will ever run if line 49 (`run: npm test`) failed
We could try with a forced-failing test

File: package.json (hunk 2 above)
Comment (thedaviddias) [reply package.json:121]: If we merge to dev, then we will change this in the beta branch.

Comment (Shinigami92) [reply package.json:121]: ok

File: .github/workflows/ci.yml (hunk 3 above)
Comment (thedaviddias) [reply .github/workflows/ci.yml:51]: If the test fails then coverage should normally not run.

**Pull request merged**